docs/user/trusted-publishers/troubleshooting.md

```diff
@@ -6,6 +6,20 @@ title: Troubleshooting
 
 # Troubleshooting
 
+## Ratelimiting
+
+PyPI currently imposes somewhat conservative restrictions on trusted
+publisher registration: no more than 20 publishers can be registered by
+a single user or IP address within a 24 hour window.
+
+This should be more than sufficient for most users (since publisher
+registration should happen rarely relative to publisher use), but maintainers
+with large numbers of projects or who access PyPI via a shared IP address
+may run into ratelimiting errors. If this happens to you, please wait 24 hours,
+try again, and then contact PyPI's admins if the problem persists.
```

Comment:
Flagging for review: this documents the current ratelimits, which are probably excessively conservative. I don't think there's any serious risk in bumping them from 20/day to 100/day, if we want to reduce the amount of maintainer interaction over this 🙂